src/app/modules/ix-forms/services/ix-formatter.service.ts

```diff
@@ -193,4 +197,16 @@ export class IxFormatterService {
     }
     return (1024 ** (this.iecUnits.indexOf(unitStr)));
   };
+
+  stringAsUrlFormatting = (value: string): string => {
```

Comment:
This doesn't look correct. Just having a parser should be enough.
I think there is a mistake in `ix-input`, where it doesn't update visible value after it has been parsed.
Please update `ix-input` instead and add a test case to it.

Reply:
It makes sense. Thanks for bringing this to my attention 👍🏻